src/planner/ompl/Planner.cpp

```diff
@@ -372,6 +374,114 @@ trajectory::InterpolatedPtr planCRRTConnect(
       std::move(_interpolator),
       _maxPlanTime);
 }
+
+//=============================================================================
+std::pair<std::unique_ptr<trajectory::Interpolated>, bool> simplifyOMPL(
+    statespace::StateSpacePtr _stateSpace,
+    statespace::InterpolatorPtr _interpolator,
+    distance::DistanceMetricPtr _dmetric,
+    constraint::SampleablePtr _sampler,
+    constraint::TestablePtr _validityConstraint,
+    constraint::TestablePtr _boundsConstraint,
+    constraint::ProjectablePtr _boundsProjector,
+    double _maxDistanceBtwValidityChecks,
+    double _timeout,
+    size_t _maxEmptySteps,
+    trajectory::InterpolatedPtr _originalTraj)
+{
```

Comment:
Nit:
- Check that `_maxEmptyChecks` is non-negative.
- Check that `_timeout` is non-negative.